dist/hidapi.podspec

```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |spec|
 
   spec.name         = "hidapi"
-  spec.version      = "0.9.0"
+  spec.version      = "<fill me up from VERSION file, before submit>"
```

Comment:
Looks like it's possible to read version from VERSION file
`spec.version = File.read('../VERSION')`
I'm going to try this during publishing 0.10.0, and if it will work, I create a PR

Reply:
sounds great